aws/table_aws_route53_zone.go

```diff
@@ -72,12 +72,24 @@ func tableAwsRoute53Zone(_ context.Context) *plugin.Table {
 				Description: "The number of resource record sets in the hosted zone.",
 				Type:        proto.ColumnType_INT,
 			},
+			{
+				Name:        "key_signing_keys",
+				Description: "The key-signing keys (KSKs) in AWS account.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getHostedZoneDNSSEC,
+			},
 			{
 				Name:        "query_logging_configs",
 				Description: "A list of configuration for DNS query logging that is associated with the current AWS account.",
 				Type:        proto.ColumnType_JSON,
 				Hydrate:     getHostedZoneQueryLoggingConfigs,
 			},
+			{
+				Name:        "dnssec_status",
+				Description: "The status of DNSSEC.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getHostedZoneDNSSEC,
```

Comment:
@anisadas Can you please verify this column value?
Don't we need `Transform` for this, since we have updated the column name?
![image](https://user-images.githubusercontent.com/38218418/120495121-0939c580-c3da-11eb-8e82-3c467f8bba42.png)